components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/SetupCmd.java

```diff
@@ -74,93 +73,70 @@
 /**
  * This class represents the "setup" command and it holds arguments and flags specified by the user.
  */
-@Parameters(commandNames = "setup", commandDescription = "setup information")
-public class SetupCmd implements GatewayLauncherCmd {
+@Parameters(commandNames = "setup", commandDescription = "setup information") public class SetupCmd
```

Comment:
Formatting issue

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/332/commits/7dec453bb0eb0d526d5b95004e51f7b012efb678